compliance/collection/inventory/convert.go

```diff
@@ -0,0 +1,110 @@
+package inventory
+
+import (
+	timestamp "github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/uuid"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+)
+
+// NodeInventoryResponseToNodeInventory converts a NodeInventory response to a native storage.NodeInventory version
+func NodeInventoryResponseToNodeInventory(response *scannerV1.GetNodeInventoryResponse) *storage.NodeInventory {
+	ni := &storage.NodeInventory{
+		NodeId:     uuid.Nil.String(),
+		NodeName:   response.GetNodeName(),
+		ScanTime:   timestamp.TimestampNow(),
+		Components: inventoryComponentsToStorageComponents(response.GetComponents()),
+		Notes:      convertNotes(response.GetNotes()),
+	}
+	return ni
+}
+
+func inventoryComponentsToStorageComponents(c *scannerV1.Components) *storage.NodeInventory_Components {
```

Comment:
the same here, you would call that on a components object, so no need for the prefix `inventoryComponents` in the name. I would call it `toStorageComponents`

Reply:
Good point, done!